storage/README.md

```diff
@@ -13,10 +13,32 @@ This library contains a entity framework for bbolt, including CRUD (Create, Read
        * bbolt utilities for getting/setting typed data from bbolt (see typed_bucket.go) 
        * Implementation of AST interfaces for reading symbols and getting symbol type information
 
+# ANTLR patches
+## Race condition
 Note: The ANTLR4 Go implementation has a race condition, detected by go -race. 
 The issue is tracked here: https://github.com/antlr/antlr4/issues/2040 
 The issue was fixed as described here: https://github.com/google/cel-go/pull/177/files
 It's not clear that this fix is required, but it appeases the race condition detector and it seems safer to have it than not.
 The fix should be reapplied after regenerating source. The next time source is regenerated, see if we can automate this
 in the generate.sh script by including the patch.
    
+## ARM 32 Patch
+A compile issue happens on arm 32 which required the following patch:
+
+```diff
+diff --git a/storage/zitiql/zitiql_parser.go b/storage/zitiql/zitiql_parser.go
+index af80e9d..8732ec4 100644
+--- a/storage/zitiql/zitiql_parser.go
++++ b/storage/zitiql/zitiql_parser.go
+@@ -1170,7 +1170,7 @@ func (p *ZitiQlParser) Start() (localctx IStartContext) {
+        p.GetErrorHandler().Sync(p)
+        _la = p.GetTokenStream().LA(1)
+ 
+-       for ((_la-3)&-(0x1f+1)) == 0 && ((1<<uint((_la-3)))&((1<<(ZitiQlParserLPAREN-3))|(1<<(ZitiQlParserBOOL-3))|(1<<(ZitiQlParserALL_OF-3))|(1<<(ZitiQlParserANY_OF-3))|(1<<(ZitiQlParserCOUNT-3))|(1<<(ZitiQlParserISEMPTY-3))|(1<<(ZitiQlParserNOT-3))|(1<<(ZitiQlParserIDENTIFIER-3)))) != 0 {
++       for ((_la-3)&-(0x1f+1)) == 0 && ((uint64(1)<<uint((_la-3)))&((1<<(ZitiQlParserLPAREN-3))|(1<<(ZitiQlParserBOOL-3))|(1<<(ZitiQlParserALL_OF-3))|(1<<(ZitiQlParserANY_OF-3))|(1<<(ZitiQlParserCOUNT-3))|(1<<(ZitiQlParserISEMPTY-3))|(1<<(ZitiQlParserNOT-3))|(1<<(ZitiQlParserIDENTIFIER-3)))) != 0 {
```

Comment:
Pretty sure we could patch this in the Antlr project.

Reply:
Yes, I'll create some backlog entries to submit some patches to ANTLR